docs/sources/alerting/manage-notifications/template-notifications/using-go-templating-language.md

```diff
@@ -71,6 +71,14 @@ The name of the annotation is {{ .Name }}, and the value is {{ .Value }}
 {{ end }}
 ```
 
+## Use a specific label value
+
+You may only want to use some specific label values in your text, or use a longer description. 
```

Comment:
```suggestion
To print a specific annotation or label you can use the `index` function.
```

Reply:
To print a specific annotation or label, use the 'index' function.